packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -73,10 +74,25 @@ const responsiveProperties = defineProperties({
     paddingRight: vars.sizes,
     marginTop: vars.sizes,
     marginBottom: vars.sizes,
-    marginLeft: vars.sizes,
-    marginRight: vars.sizes,
-    width: { ...vars.sizes, '100%': '100%', 'max-content': 'max-content' },
-    height: { ...vars.sizes, '100%': '100%' },
+    marginLeft: { ...vars.sizes, $auto: 'auto' },
+    marginRight: { ...vars.sizes, $auto: 'auto' },
+    width: {
+      ...vars.sizes,
+      '100%': '100%',
+      'max-content': 'max-content',
+      $screen25: '25vw',
+      $screen50: '50vw',
+      $screen75: '75vw',
+      $screen100: '100vw',
+    },
+    height: {
+      ...vars.sizes,
+      '100%': '100%',
+      $screen25: '25vh',
+      $screen50: '50vh',
+      $screen75: '75vh',
+      $screen100: '100vh',
```

Comment:
Are these values that we would use often? For me, these seem like style values that would only be used once or twice in an app